src/core/scopes.cc

```diff
@@ -1481,6 +1499,11 @@ int32_t scopes::process(const waflz_pb::enforcement** ao_enf,
                         WAFLZ_PERROR(m_err_msg, "performing rqst_ctx::append_rqst_info for acl");
                         return WAFLZ_STATUS_ERROR;
                 }
+                // -------------------------------------------------
+                // Set config properties
+                // -------------------------------------------------
+                l_event->set_acl_config_id(l_acl->get_id());
```

Comment:
This can be moved to acl process call i think, to prevent setting it twice seperately in scopes->process